pkg/apis/core/validation/validation_test.go

```diff
@@ -6749,6 +6752,14 @@ func TestValidatePodSpec(t *testing.T) {
 			RuntimeClassName: utilpointer.StringPtr("valid-sandbox"),
 			Overhead:         core.ResourceList{},
 		},
+		{
```

Comment:
Can we also add a test case to validate we can't change the field?

Reply:
fixed